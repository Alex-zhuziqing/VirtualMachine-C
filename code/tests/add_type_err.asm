FUNC LABEL 0
    MOV STK A VAL 1
    MOV STK B VAL 2
    ADD STK A STK B
    RET
