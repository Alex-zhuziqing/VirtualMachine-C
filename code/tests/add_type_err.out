Cannot add the values outside the registers.
